geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/NullRedisSet.java

```diff
@@ -122,29 +117,29 @@ private int doSetOpWhileLocked(SetOp setOp, CommandHelper helper,
     }
   }
 
-  private Set<ByteArrayWrapper> computeSetOp(SetOp setOp,
-      ArrayList<Set<ByteArrayWrapper>> nonDestinationSets,
-      CommandHelper helper,
-      RedisKey destination) {
-    Set<ByteArrayWrapper> result = null;
+  private Set<byte[]> computeSetOp(SetOp setOp, List<Set<byte[]>> nonDestinationSets,
+      CommandHelper helper, RedisKey destination) {
+    ObjectOpenCustomHashSet<byte[]> result = null;
     if (nonDestinationSets.isEmpty()) {
       return emptySet();
     }
-    for (Set<ByteArrayWrapper> set : nonDestinationSets) {
+    for (Set<byte[]> set : nonDestinationSets) {
```

Comment:
This didn't start with your changes, but these set operations may happen on very large sets - we will probably want to eliminate as much set copying as possible before making these things supported commands.